src/modules/runs/components/workflowRuns/WorkflowRunTimeline.tsx

```diff
@@ -560,3 +561,25 @@ const WorkflowRunTimeline = () => {
 };
 
 export default WorkflowRunTimeline;
+
+/**
```

Comment:
I think this part was removed because of refactoring from `statusTimeline.tsx` (#49). But happy to remove it if this approach is too naive